examples/util/fill.rs

```diff
@@ -70,7 +70,7 @@ mod platform {
         }
     }
 
-    pub fn fill_window(window: &dyn Window) {
+    pub fn fill_window(window: &dyn Window, color: u32) {
```

Comment:
I'd suggest to just add a new function `fill_window`, which calls `fill_window_with_color`, which uses the default color we're using, thus way you don't have to change any example at all, except the child one.